### PR TITLE
Export Job to IRSA

### PR DESCRIPTION
**Pull request by EliseCastle23**
changing the export job to use environment variables and IRSA instead of access keys

### Improvements
changing the batch export job to use IRSA instead of access keys. Also, changing the job to use environment variables to read in the bucket name for the batch export job. 

### Deployment changes
Manifest.json files MUST be changed to use the new "batch-export-sa" service account and the "BUCKET" environment variable must be set to the batch-export-g3auto secret with the "bucket_name" key. You also no longer need to mount the batch-export-g3auto secret as this is now deprecated (it is replaced by the batch-export-g3auto configmap). 
```
"serviceAccountName": "batch-export-sa"...


      - name: BUCKET
        valueFrom:
           configMapKeyRef:
            name: batch-export-g3auto
            key: bucket_name

Remove the following:   
       "volumeMounts": [
          {
            "name": "batch-export-creds-volume",
            "readOnly": true,
            "mountPath": "/batch-export-creds.json",
            "subPath": "config.json"
          }
        ],
      },
      "volumes": [
        {
          "name": "batch-export-creds-volume",
          "secret": {
            "secretName": "batch-export-g3auto"
          }
        }
      ],

### CODE REVIEW

File: export_job/export_zip_from_manifest_job.py [hunk 1]
```diff
@@ -129,6 +123,9 @@ def fail(error_message=DEFAULT_ERROR_MESSAGE):
 if __name__ == "__main__":
     access_token = os.environ["ACCESS_TOKEN"]
     hostname = os.environ["GEN3_HOSTNAME"]
+    bucket_name = os.environ["BUCKET"]
+    print("Printing Bucket Name")
+    print(bucket_name)
```
Comment (mfshao) [inline export_job/export_zip_from_manifest_job.py:128]: these are for debugging only right? I think we should remove them

File: export_job/export_zip_from_manifest_job.py [hunk 2]
```diff
@@ -129,6 +123,9 @@ def fail(error_message=DEFAULT_ERROR_MESSAGE):
 if __name__ == "__main__":
     access_token = os.environ["ACCESS_TOKEN"]
     hostname = os.environ["GEN3_HOSTNAME"]
+    bucket_name = os.environ["BUCKET"]
```
Comment (mfshao) [inline export_job/export_zip_from_manifest_job.py:126]: maybe consider failing the job if this env var is not set?

Comment (EliseCastle23) [reply export_job/export_zip_from_manifest_job.py]: That is a great idea! I just updated the code to reflect this!

**Review by mfshao — APPROVED** [review]
👍

**Pull request merged**